test/test-errortracker.js

```diff
@@ -249,18 +246,118 @@ describe('Test how server responds to requests/behave', function() {
   });
 
   it('Should ignore debug errors', function() {
-    // adjust query parameters to mock this case
     randomVal = 0.00000000000000001;
     query.a = 0;
     query.ca = 1;
     query['3p'] = 0;
-    query.s = 'exception';
+    query.s = '  at new vi (https://cdn.ampproject.org/rtv/031496877433269/v0.js:297:149)';
     query.debug = 0;
     query.r = 'referer';
     query.m = 'message';
     return chai.request(app).get('/r').query(query).then(function(res) {
       expect(res).to.have.status(statusCodes.NO_CONTENT);
     });
   });
+
+  it('Should not log exceptions with only invalid stacktraces', function() {
+    randomVal = 0.00000000000000001;
+    query.a = 0;
+    query.ca = 1;
+    query['3p'] = 0;
+    query.s = 'exception@file.js';
+    query.debug = 1;
+    query.r = 'referer';
+    query.m = 'message';
+    return chai.request(app).get('/r').query(query).then(function(res) {
+      throw new Error('Unreachable');
+    }, function(res) {
+      /** chai-http errors with handling > 299 status codes hence
+       * errors can only be asserted in the catch block which
+       * modifies anatomy of response
+       * object. More information at https://github.com/chaijs/chai-http/issues/75.
+       * This is a hack and once the package
+       * has been updated is subject to change
+       */
```

Comment:
Use multiple `//` comments for documenting code blocks. Save JSDoc `/** */` for classes, functions, etc.